app/modules/metadata.js

```diff
@@ -53,7 +54,15 @@ export const checkVersion = () => async (dispatch: DispatchType) => {
 
     if (shouldUpdate) {
       showError(
-        `Your wallet is out of date! Please download the latest version from ${DOWNLOAD_LINK}`
+        <div>
+          Your wallet is out of date! Please download the latest version from{' '}
+          <div
+            className="notification-download-link"
+            onClick={() => shell.openExternal(NEON_WALLET_RELEASE_LINK)}
+          >
+            {NEON_WALLET_RELEASE_LINK}
+          </div>
+        </div>
```

Comment:
Perhaps use `<ConditionalLink>` which already opens an external browser (in #1746):
```jsx
<ConditionalLink href={NEON_WALLET_RELEASE_LINK}>
  {NEON_WALLET_RELEASE_LINK}
</ConditionalLink>
```
Pros:
a) Readable and concise
b) Link would behave and style as one
c) Optional `tooltip` attr

Reply:
👍 awesome